src/lib/main.js

```diff
@@ -16,16 +18,16 @@ XPCOMUtils.defineLazyModuleGetter(this, 'console',
 XPCOMUtils.defineLazyModuleGetter(this, 'CustomizableUI',
   'resource:///modules/CustomizableUI.jsm');
 
-var EXPORTED_SYMBOLS = ['Main']; // eslint-disable-line no-unused-vars
+const EXPORTED_SYMBOLS = ['Main']; // eslint-disable-line no-unused-vars
 
-var onTabSelect = function() { console.log('onTabSelect'); };
-var onTabOpen = function() { console.log('onTabOpen'); };
-var onTabClose = function() { console.log('onTabClose'); };
+const onTabSelect = function() { console.log('onTabSelect'); };
```

Comment:
> @6a68 I personally am not a fan of this anonymous function bid'ness. I think it makes the stack dumps a bit harder to read.

Yeah, it's really nice to have function.name. The browser devtools give you line numbers and make this a lot nicer, but still.

> I can dig into some ESLint rules and see if there is an existing rule (or custom rule) to not allow var foo = function () {}.

Sounds great! Disallowing function expressions has got to be in there; name hoisting is definitely weirder than double-equals comparators.